pkg/datarepair/repairer/repairer.go

```diff
@@ -15,35 +15,36 @@ import (
 	segment "storj.io/storj/pkg/storage/segments"
 )
 
-// Repairer is the interface for the data repairer
```

Comment:
Do we need all those changes in `repairer.go` when `NewRepairer` is not used anymore? Now it doesn't have to be accessible outside package.